### ISSUE TITLE
About BasicVSR++ pretrained models on track1

### ISSUE THREAD

**Issue by meobach** [open-mmlab/mmediting#748]
Hi,
Can you tell me the validation set you use during the training BasicVSR++ on track1? I want to fine-tune it, So It is very useful for me if I know which set for validation in the training phase.
Thank you so much

**Comment by ckkelvinchan** [comment]
Hello, I used the official validation set in NTIRE 2021.

**Comment by meobach** [comment]
Thank you,
I know last year's competition which has 200 videos for training, 20 videos for the validation phase, and 20 videos for the testing phase, but the validation set and test set do not release the ground true video, so how can you validate your models during the training time?  you just have trained your models on 200 videos with 600k iteration without any validation set. or you spend some videos in 200 training videos for validation.
Sorry about my long question but I am in this year competition and I want to know the exactly how you split data for training and choose correctly validation set

**Comment by ckkelvinchan** [comment]
The released model used all 200 training videos. If I remembered correctly, we have the ground-truths of the validation set last year.

**Comment by meobach** [comment]
Thank you for your help so much! ^^

**Comment by meobach** [comment]
Hello, Can you share with me the training tip and config you have used in last year's competition? In this year's competition, I have fine-tuned the pre-trained models with new data(the old 200 videos from last year and 40 additional videos). But the accuracy is slightly down, I have used the batch size for each GPU as 1 and so the total batch size for each iteration is 4, the number of frames I chose is 17. 
Thank you!

**Comment by ckkelvinchan** [comment]
The performance depends on the batch size and the number of frames during training. I used batch size = 8 and number of frames = 30 last year. 

It is hard to prevent performance drop if a smaller batch size and shorter sequence are used for training. One thing you can try is to fine-tune smaller models to a larger model. We use this trick throughout the competition. It may save you some time.

**Comment by meobach** [comment]
Hi @ckkelvinchan , thank you for your response.
What "fine-tune smaller models to a larger model" mean? I am not really clear about it.

**Comment by ckkelvinchan** [comment]
Initially, we train a smaller model (channel=64, blocks=7) for explorational purpose. We finally train the large model by using the weights as initialisation. This can save you some time as you don't need to train from scratch

**Comment by meobach** [comment]
Thank for so much for your helpful trick!

**Comment by yytdfc** [comment]
> 

can you give some details about how to initialize large model from a small one, just copy weights from the smaller model? what about other weights?

**Comment by yytdfc** [comment]
> Initially, we train a smaller model (channel=64, blocks=7) for explorational purpose. We finally train the large model by using the weights as initialisation. This can save you some time as you don't need to train from scratch

can you give some details about how to initialize large model from a small one, just copy weights from the smaller model? what about other weights?

**Comment by ckkelvinchan** [comment]
Assuming we are scaling up from 64 channels to 128 channels. Then a convolutional layer would be from `64x64x3x3` to `128x128x3x3`. In this case, you can repeat the weights and divide the weights by 4. And if you increase the number of blocks, then you can simply copy the weights from the previous blocks.

I did not try training it from scratch and therefore I cannot be sure how much improvements we can obtain.

**Comment by yytdfc** [comment]
> Assuming we are scaling up from 64 channels to 128 channels. Then a convolutional layer would be from `64x64x3x3` to `128x128x3x3`. In this case, you can repeat the weights and divide the weights by 4. And if you increase the number of blocks, then you can simply copy the weights from the previous blocks.
> 
> I did not try training it from scratch and therefore I cannot be sure how much improvements we can obtain.

Thank you ! 👍
